File: bai 6/app/src/main/java/com/example/bth6_xulysukien_tinhtong/MainActivity.java
package com.example.bth6_xulysukien_tinhtong;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void XuLyCong(View view){
        EditText edtSo1 = findViewById(R.id.edtSo1);
        EditText edtSo2 = findViewById(R.id.edtSo2);
        TextView txtKetQua = findViewById(R.id.txtKetQua);

        String strA = edtSo1.getText().toString();
        String strB = edtSo2.getText().toString();

//        if (strA.isEmpty() || strB.isEmpty()) {
//            txtKetQua.setText("Nhập đủ 2 số!");
//            return;
//        } hàm kiểm tra thêm

        int a = Integer.parseInt(strA);
        int b = Integer.parseInt(strB);

        txtKetQua.setText("Kết quả: " + (a + b));
    }
}

File: Bai4/app/src/main/java/com/example/bth4_linearlayout_tong2so/MainActivity.kt
package com.example.bth4_linearlayout_tong2so

import android.os.Bundle
import android.view.View
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    // Xử lý cộng
    fun XuLyCong(v: View) {

        val editTextSo1 = findViewById<EditText>(R.id.edtSo1)
        val editTextSo2 = findViewById<EditText>(R.id.edtSo2)

        val soThu1 = editTextSo1.text.toString()
        val soThu2 = editTextSo2.text.toString()

        val soA = soThu1.toFloat()
        val soB = soThu2.toFloat()

        val tong = soA + soB

        val editTextKQ = findViewById<EditText>(R.id.edtKetQua)

        val chuoiKQ = tong.toString()

        editTextKQ.setText(chuoiKQ)
    }

    // Xử lý trừ
    fun XuLyTru(v: View) {

        val editTextSo1 = findViewById<EditText>(R.id.edtSo1)
        val editTextSo2 = findViewById<EditText>(R.id.edtSo2)

        val soA = editTextSo1.text.toString().toFloat()
        val soB = editTextSo2.text.toString().toFloat()

        val tong = soA - soB

        val editTextKQ = findViewById<EditText>(R.id.edtKetQua)

        editTextKQ.setText(tong.toString())
    }

    // Nhân
    fun XuLyNhan(v: View) {

        val soA = findViewById<EditText>(R.id.edtSo1).text.toString().toFloat()
        val soB = findViewById<EditText>(R.id.edtSo2).text.toString().toFloat()

        val kq = soA * soB

        findViewById<EditText>(R.id.edtKetQua).setText(kq.toString())
    }

    // Chia
    fun XuLyChia(v: View) {

        val soA = findViewById<EditText>(R.id.edtSo1).text.toString().toFloat()
        val soB = findViewById<EditText>(R.id.edtSo2).text.toString().toFloat()

        val kq = soA / soB

        findViewById<EditText>(R.id.edtKetQua).setText(kq.toString())
    }
}